test/dlc_tests/ec2/test_efa.py

```diff
@@ -0,0 +1,60 @@
+import os
+
+import pytest
+
+from test.test_utils import CONTAINER_TESTS_PREFIX, LOGGER, is_tf2
+from test.test_utils.ec2 import get_ec2_instance_type
+
+EFA_SCRIPT = os.path.join(CONTAINER_TESTS_PREFIX, "test_efa_sanity.py")
+
+EFA_EC2_GPU_INSTANCE_TYPE = get_ec2_instance_type(default="p3dn.24xlarge", processor="gpu")
+
+
+@pytest.mark.parametrize("ec2_instance_type", EFA_EC2_GPU_INSTANCE_TYPE, indirect=True)
+def test_sanity(training, ec2_connection, region, gpu_only, py3_only):
```

Comment:
Please add `@pytest.mark.model` and `@pytest.mark.integration` here.

Reply:
added